web-app/utils/squidex-api.js

```diff
@@ -0,0 +1,48 @@
+import axios from 'axios'
+import qs from 'qs'
+
+// App config file with API secrets
+import appConfig from  'app.config'
+
+const {
+  squidex: {
+    appName,
+    clientId,
+    clientSecret,
+    scope
+  }
+} = appConfig;
+
+
+const authenticate = function () {
+  return axios.post(
+    'https://cloud.squidex.io/identity-server/connect/token',
```

Comment:
@smoliver will there also be a credential set/api token for an api-staging? Just wondering if another dev ever has to make changes to prevent it from effecting production stuff.